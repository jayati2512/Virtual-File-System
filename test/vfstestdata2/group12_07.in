createvfs time 2048
mountvfs time
makedir /origin/ original
makedir / download
makedir /download/ folder1
deletedir /download/folder1/
addfile /download/ file1.txt /usr/include/
addfile /origin/original/ file3.txt /usr/include/
removefile /download/ file1.txt
removefile /download/ file1.txt
addfile /download/ file0.txt /usr/include/
movedir /origin/original/ /download/
listdir /download/ 0 /usr/inlude/list.txt
addfile /download/ file2.txt /usr/include/
updatefile /download/file2.txt /usr/include/list2.txt
movefile /download/file2.txt /orign/original/file3.txt
exportfile /download/file2.txt /usr/include/
searchfile file0 output.txt
unmountvfs demo
