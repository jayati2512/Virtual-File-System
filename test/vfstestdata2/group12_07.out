createvfs_SUCCESS
mountvfs_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
deletedir_SUCCESS
addfile_SUCCESS
addfile_SUCCESS
removefile_SUCCESS
removefile_FAILURE CANNOT_FIND_SPECIFIED_FILE
addfile_SUCCESS
movedir_SUCCESS
listdir_SUCCESS
addfile_SUCCESS
updatefile_SUCCESS
movefile_SUCCESS
exportfile_SUCCESS
searchfile_SUCCESS
unmount_SUCCESS
